### PR TITLE
monitor: split governance monitor into serveral methods

### PR DESCRIPTION
**Pull request by yutianwu**
### Description

This pr will refactor governance monitor and split governance monitor into several methods to make the code more clean.

### Changes

Notable changes:
* refactor governance monitor

### CODE REVIEW

File: service/apiserver/server.go [hunk 1]
```diff
@@ -21,7 +20,7 @@ func Run(configFile string) error {
 	ctx := svc.NewServiceContext(c)
 	handler.RegisterHandlers(server, ctx)
 
-	fmt.Printf("Starting server at %s:%d...\n", c.Host, c.Port)
+	logx.Infof("Starting server at %s:%d...\n", c.Host, c.Port)
```
Comment (forcodedancing) [inline service/apiserver/server.go:23]: I also change this in another pr. lol

**Review by KeefeL — APPROVED** [review]
lgtm

File: service/monitor/monitor/monitor_governance_blocks.go [hunk 2]
```diff
@@ -251,34 +208,21 @@ func (m *Monitor) MonitorGovernanceBlocks() (err error) {
 				return fmt.Errorf("unpack GovernanceAssetPausedUpdate failed, err: %v", err)
 			}
 			// set up database info
-			l1EventInfo := &L1EventInfo{
+			l1EventInfo := &L1Event{
```
Comment (forcodedancing) [inline service/monitor/monitor/monitor_governance_blocks.go:211]: `l1EventInfo` can be renamed to `l1Event`. However, it is small, no bi deal.

File: service/monitor/monitor/types.go [hunk 3]
```diff
@@ -95,7 +95,7 @@ var (
 	governanceLogAssetPausedUpdateSigHash     = crypto.Keccak256Hash(governanceLogAssetPausedUpdateSig)
 )
 
-type L1EventInfo struct {
+type L1Event struct {
```
Comment (forcodedancing) [inline service/monitor/monitor/types.go:98]: Regroup the code?
```
	EventTypeNewPriorityRequest = 0
	EventTypeCommittedBlock     = 1
	EventTypeVerifiedBlock      = 2
	EventTypeRevertedBlock      = 3

	EventNameNewAsset              = "NewAsset"
	EventNameNewGovernor           = "NewGovernor"
	EventNameNewAssetGovernance    = "NewAssetGovernance"
	EventNameValidatorStatusUpdate = "ValidatorStatusUpdate"
	EventNameAssetPausedUpdate     = "AssetPausedUpdate"

	EventTypeAddAsset              = 4
	EventTypeNewGovernor           = 5
	EventTypeNewAssetGovernance    = 6
	EventTypeValidatorStatusUpdate = 7
	EventTypeAssetPausedUpdate     = 8
```

**Review by forcodedancing — APPROVED** [review]

**Pull request merged**